package.json

```diff
@@ -71,7 +71,7 @@
     "babel-core": "^6.24.1",
     "ember-cli-babel": "^6.8.2",
     "ember-compatibility-helpers": "^1.2.0",
-    "ember-maybe-import-regenerator": "^0.1.5"
+    "ember-maybe-import-regenerator": "^0.1.6"
```

Comment:
Listing a dependency in `dependencies` and `devDependencies` doesn't really make sense. The correct fix IMO, would be removing it from `devDependencies` and raising the version here in `dependencies` to ^0.1.6`.

Reply:
This is a great point. I wasn't really sure why it was duplicated. I'll try removing it from `devDependencies` to see what happens.

BTW are the failed tests normal? As far as I can tell they also pass sometimes 🕵

Reply:
The failing tests seem to be flaky and not related to this PR.